lib/anydocs/processAnyDocument.js

```diff
@@ -0,0 +1,33 @@
+const Client = require('../client/constructor');
+const path = require('path');
+const fs = require('fs');
+/**
+ * Process any document and extract all the fields from it. https://docs.veryfi.com/api/anydocs/process-%E2%88%80-doc/
+ * @example
+ * veryfi_client.process_any_document('file/path','blue_print')
+ *
+ * @memberof Client
+ * @param {String} file_path Path on disk to a file to submit for data extraction
+ * @param {String} template_name name of the extraction templates.
+ * @param {number} max_pages_to_process The number of pages to process for the document. The limit is 50 pages per document.
+ * @param {Object} kwargs Additional request parameters
+ * @returns {JSON} Data extracted from the document
+ */
+Client.prototype.process_any_document = async function (
```

Comment:
maybe let's just have one function to read file from path and send a multipart?
So it replaces this one and process_any_document_from_buffer
I see no reason to have base64 for example as it's slower